config/telemetry.go

```diff
@@ -318,7 +318,7 @@ func (c OTLPExportersMetricsTelemetryConfig) NewExporter(ctx context.Context) (s
 	return sdkmetric.NewPeriodicReader(exporter), nil
 }
 
-type LogTelemetryConfiguration struct {
+type LogTelemetryConfig struct {
```

Comment:
Ignore this change. There is an inconsistency in config struct naming. Will address it separately.